docs/source/examples/debug_output.txt

```diff
@@ -18,8 +18,8 @@ and these constants relaxed:
 > Trying with relaxed constraints:
 
 Solves with these constraints relaxed:
-   1: 3500% relaxed, from 3*x**-1 <= 1
-                       to 3*x**-1 <= 36
+   1: 3500% relaxed, from 1 [yd] <= 1
```

Comment:
it looks like the relaxation has messed up this output...the old one is the correct thing to do here, I think!